.github/workflows/help_wanted_notify.yml

```diff
@@ -1,4 +1,4 @@
-name: notify social media (help wanted)
+name: Notify Discord about "help wanted" issues
 
 on:
   issues:
```

Comment:
I vaguely remember the way the webhook needed to be formatted being slightly obscure and that's probably what's making this not run properly but won't be a block to merging.

Reply:
I'll put up a separate PR to try and fix this!